framework/src/Volo.Abp.EntityFrameworkCore/Volo/Abp/EntityFrameworkCore/AbpDbContext.cs

```diff
@@ -486,6 +486,7 @@ protected virtual void ApplyAbpConceptsForDeletedEntity(EntityEntry entry)
 
             entry.Reload();
             entry.Entity.As<ISoftDelete>().IsDeleted = true;
+            SetDeletionAuditProperties(entry);
             entry.State = EntityState.Modified;
```

Comment:
this line[489] will cause full entity update, what is the #10737 wants to fix.
Remove this line?